test/unit/enrichment/is-state-aid.test.js

```diff
@@ -0,0 +1,44 @@
+const { CS } = require('../../../app/constants/schemes')
+const { EXQ00 } = require('../../../app/constants/fund-codes')
+const { SOS228, SOS229 } = require('../../../app/constants/account-codes')
+
+const { isStateAid } = require('../../../app/enrichment/is-state-aid')
+
+let invoiceLine
+
+describe('is state aid', () => {
+  beforeEach(() => {
+    invoiceLine = JSON.parse(JSON.stringify(require('../../mocks/payment-requests/invoice-line')))
+  })
+
+  test.each([
+    [EXQ00, SOS228, true],
+    [EXQ00, SOS229, true],
+    [EXQ00, 'SOS273', false],
+    ['ERD14', SOS228, false],
+    ['ERD14', SOS229, false],
+    ['ERD14', 'SOS273', false],
+    ['ERD14', undefined, false],
+    [undefined, SOS228, false],
+    [undefined, SOS229, false],
+    [undefined, 'SOS273', false],
+    [undefined, undefined, false],
+    [null, null, false],
+    [null, SOS228, false],
+    [null, SOS229, false],
+    [null, 'SOS273', false],
+    [null, undefined, false],
+    [EXQ00, null, false],
+    ['ERD14', null, false]
+  ])('should return %s for fund code %s and account code %s', (fundCode, accountCode, expected) => {
```

Comment:
What does `%s` mean here?

Reply:
Substitutes the arguments above in as text in order